packages/manager/apps/web-hosting/src/pages/managedWordpress/ManagedWordpressResource/import/steps/ImportForm.component.tsx

```diff
@@ -0,0 +1,137 @@
+import { ManagerButton, Subtitle } from '@ovh-ux/manager-react-components';
+import {
+  OdsFormField,
+  OdsInput,
+  OdsPassword,
+  OdsText,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_COLOR,
+  ODS_INPUT_TYPE,
+  ODS_TEXT_PRESET,
+} from '@ovhcloud/ods-components';
+import { NAMESPACES } from '@ovh-ux/manager-common-translations';
+import { useTranslation } from 'react-i18next';
+import { Controller, useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useState } from 'react';
+import { zForm } from '@/utils/formSchemas.utils';
+
+export default function ImportForm() {
+  const { t } = useTranslation([NAMESPACES.FORM]);
+  const [step, setStep] = useState(1);
+
+  const {
+    control,
+    handleSubmit,
+    formState: { isValid, errors, isDirty },
+  } = useForm({
+    defaultValues: {
+      url: '',
+      password: '',
+      login: '',
+    },
+    mode: 'onChange',
+    resolver: zodResolver(zForm(t).ADD_SITE_FORM_SCHEMA),
+  });
+
+  const onStep1Submit = () => {
+    setStep(2);
+  };
+
+  return (
+    <>
+      {step === 1 && (
+        <form
+          className="flex flex-col items-start w-full md:w-1/2 gap-4 mt-4"
+          onSubmit={handleSubmit(onStep1Submit)}
+        >
+          <OdsText preset={ODS_TEXT_PRESET.span}>
+            {t(`${NAMESPACES.FORM}:mandatory_fields`)}
+          </OdsText>
+
+          <Subtitle className="mt-4 mb-4">Url de connexion</Subtitle>
+          <Controller
+            name="url"
+            control={control}
+            render={({ field: { name, value, onBlur, onChange } }) => (
+              <OdsFormField className="w-full" error={errors?.url?.message}>
+                <label slot="label">{t("Url d'administration du site")}*</label>
```

Comment:
```suggestion
                <label slot="label">{t("Url d'administration du site")}*</label>
```
Are you sure for this translation key ?